### PR TITLE
 SW-3089 : Added client log router

### PR DESCRIPTION
**Pull request by suriyakumar2211**

### CODE REVIEW

**Review by AkshathaSar — APPROVED** [review]

**Review by anujsinghaniya — APPROVED** [review]

File: api/clientlogger.go [hunk 1]
```diff
@@ -17,29 +16,36 @@ type ClientLogger struct {
...
 			for _, log := range requestModel {
-				jsonStr, err := json.Marshal(log)
-				if err == nil {
-					fmt.Println("Log captured at::", log.Time, " ", string(jsonStr))
-				} else {
-					fmt.Println("Error while marshalling the log data:::", err)
-				}
+				event := getLevel(log)
+				event.Msg(log.Message + `, Application Name: ` + log.AppName +
+					`, UserID: ` + log.UserID + `, Url: ` + log.Url +
```
Comment (anujsinghaniya) [inline api/clientlogger.go:35]: you need to have this as keys.

**Review by anujsinghaniya — APPROVED** [review]
I am approving this but

File: api/router.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+// GetRouter is used to get the router configured with the middlewares and the routes
+func GetRouter(middlewaresList ...gin.HandlerFunc) *gin.Engine {
+
+	router := gin.New()
+	router.Use(middlewaresList...)
+	router.Use(gin.Recovery())
+	allowedOrigins := configs.GetClient().GetStringSliceD(constants.ApplicationConfig, constants.AllowedOrigins, []string{})
+	cors := middlewares.InitCors(allowedOrigins)
+	router.Use(rate_limiter.GetRateLimiterMiddleware(), cors.CORSMiddleware(), auth.GetAuthMiddleware(), auth.NonTradeTokenAuthMW(BBELogger))
+	router.GET(constants.SwaggerRoute, ginSwagger.WrapHandler(swaggerFiles.Handler))
```
Comment (digangelone) [inline api/router.go:37]: Please use already set value  ExpiresAt by common authenticator

File: api/clientlogger.go (hunk 1 above)
Comment (suriyakumar2211) [reply api/clientlogger.go]: Changes added @anujsinghaniya

File: api/router.go (hunk 2 above)
Comment (suriyakumar2211) [reply api/router.go:37]: ExpiresAt is only set when authentication is success, in the authentication failure state "ExpiresAt" is not available @digangelone

**Pull request merged**